packages/react-components/src/molecules/TagList.tsx

```diff
@@ -21,7 +21,6 @@ const listStyles = css({
 });
 
 const normalListItemStyles = css({
-  marginBottom: `${12 / perRem}em`,
```

Comment:
The working group card might need a touch, the tags look too close to the text "Last updated" now 
<img width="1343" alt="Screenshot 2024-03-25 at 14 24 29" src="https://github.com/yldio/asap-hub/assets/16595804/d18e4150-cc7d-4e86-9e3d-5b485e46f954">